### DISCUSSION TITLE
Doubt: in Equal area or equal angle projection while plotting the incomplete pole figure

### DISCUSSION THREAD

**Discussion by AmitS07** [mtex-toolbox/mtex#588] [Ask Anything] [answered]
Dear Prof. Ralf,

I have a small question. 
While plotting the recalculated pole figures we have the option of using either equal area or equal angle projection. However, how does MTEX reads the experimental incomplete pole figure data? Is it in equal area or equal angle mode? 
Will choosing any one option change the ODF calculation using ‘calcODF option’.
I think, while plotting incomplete pole figure in MTEX, it is by default considering equal area projection. could it will be possible to change this into equal angle projection?

Thanking You
Amit

**Comment by ralfHielscher** [comment, marked as answer]
Hi Amit,

equal area or equal angle projection do matter only during plotting. When importing such data you are usually given polar angles for the diffraction directions. 

Read here https://mtex-toolbox.github.io/SphericalProjections.html to learn how to change the spherical projection during plotting.

Ralf.

**Reply by AmitS07** [reply]
Dear Prof. Ralf,

Thank you for the quick answer. So, the ODF calculation is not affected by either equal area or equal angle as we are using the polar angles viz. alpha tilt and beta rotation for computing ODF. 

I have a quick question regarding annotation of miller indices on pole figures. I managed to get the pole figure in equal angle mode by using the command ‘eangle’. However, when I am trying to annotate some miller indices on to the pole figure it does that in equal area mode. Is there a possible way to solve it? 

plotPDF(odf,Miller(1,1,1,CS)],'projection','eangle','contour','antipodal','complete','linewidth',3,'colorrange',[1 4.4],'complete','upper');

mod1 = orientation.byEuler(90*degree,35*degree,45*degree,CS,SS);
annotate (mod1,'marker','s','MarkerSize',15,'MarkerFaceColor','k', 'label','A','color','r')


Thanks in advance
-----------
Amit

**Reply by ralfHielscher** [reply]
> However, when I am trying to annotate some miller indices on to the pole figure it does that in equal area mode. 

That is not true. It takes always the correct projection.
